### PR TITLE
transaction fee math

### PR DESCRIPTION
**Pull request by ashleynguyen27**
[//]: # "These comments meant for your reference, they are invisible and don't need to be deleted"

## What's new in this PR?
Added the transaction fee
[//]: # '####### YOUR ANSWER BELOW ###########'
[//]: # '############## END ##################'

### How to Review
Step 4 & 5 of onboarding: the transaction fee should be calculated correctly 
transaction fee = amount * 2.9% + $0.30
[//]: # 'The order in which to review files and 
what to expect when testing locally'
[//]: # '####### YOUR ANSWER BELOW ###########'
[//]: # '############## END ##################'

### Related PRs

[//]: # "Optional - related PRs you're waiting on
/ PRs that will conflict, etc"
[//]: # '####### YOUR ANSWER BELOW ###########'
[//]: # '############## END ##################'

### Next steps

[//]: # "What doesn't work yet, what's NOT in this 
PR/has to be done "
[//]: # '####### YOUR ANSWER BELOW ###########'
[//]: # '############## END ##################'

### Screenshots

[//]: # '#### YOUR SCREENSHOTS BELOW ########'
[//]: # '############## END ##################'

### Design Status
HIFI
[//]: # 'If this is a frontend PR, what is the expected 
status of the UI in this PR (lo, mid, high- fi?'
[//]: # '####### LOFI/MIDFI/HIFI ###########'
[//]: # '############## END ##################'

CC: @dfangshuo

### CODE REVIEW

File: src/screens/onboarding/steps/PaymentDetailsStep.js [hunk 1]
```diff
@@ -66,6 +67,9 @@ class PaymentDetailsStep extends React.Component {
   render() {
     const { owner, errors, onBack, handleChange } = this.props;
     const { displayUnmarkedDividend } = this.state;
+    const transactionFee =
+      (owner.numberOfShares * SHARE_PRICE + 0.3) / (1 - 0.029) -
+      owner.numberOfShares * SHARE_PRICE;
```
Comment (dfangshuo) [inline src/screens/onboarding/steps/PaymentDetailsStep.js:72]: don't put math in render function. Makes it hard to keep track. Please move this into a paypalUtils.js

File: src/screens/onboarding/steps/PaymentDetailsStep.js [hunk 2]
```diff
@@ -161,11 +165,24 @@ class PaymentDetailsStep extends React.Component {
               <div className="payment-summary-qty">
                 QTY: {owner.numberOfShares}
               </div>
+              <div className="flex justify-space-between">
+                <div className="left payment-summary-shares">
+                  Transaction Fee{' '}
```
Comment (dfangshuo) [inline src/screens/onboarding/steps/PaymentDetailsStep.js:170]: is this space at the end for UI reasons?

File: src/screens/onboarding/steps/PaymentDetailsStep.js [hunk 3]
```diff
@@ -161,11 +165,24 @@ class PaymentDetailsStep extends React.Component {
...
               <hr className="payment-summary-hr" />
               <div className="flex justify-space-between">
                 <div className="left payment-summary-total">Total</div>
                 <div className="right payment-summary-total">
-                  ${owner.numberOfShares * SHARE_PRICE}.00
+                  $
+                  {(
+                    owner.numberOfShares * SHARE_PRICE +
+                    transactionFee
+                  ).toFixed(2)}
```
Comment (dfangshuo) [inline src/screens/onboarding/steps/PaymentDetailsStep.js:185]: again, leave such calculations out of the render() function, refactor this into a helper function that takes int numberOfshares

File: src/screens/onboarding/steps/PaymentDetailsStep.js (hunk 2 above)
Comment (ashleynguyen27) [reply src/screens/onboarding/steps/PaymentDetailsStep.js:175]: yes

**Pull request merged**